posthog/queries/funnels/test/test_funnel_strict.py

```diff
@@ -28,6 +28,13 @@ class TestFunnelStrictStepsBreakdown(ClickhouseTestMixin, funnel_breakdown_test_
 
     maxDiff = None
 
+    def test_basic_funnel_default_funnel_days_breakdown_event(self):
```

Comment:
I spent a couple hours trying to fix this and couldn't

These tests are now failing because https://github.com/PostHog/posthog/pull/12137/files#diff-61fc3375737cd6c09187397b2efc67245ec0d1ef1fd8fd19e685b554d8458765L1041 -- these tests were always testing ClickhouseFunnel, not the specific funnel that was passed. We need to solve this, but outside the scope of this PR